### PR TITLE
Add custom filters

### PR DESCRIPTION
**Pull request by mnshaw**
Add ability to add more filters if there is a need to filter more words.

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/test-infra/397)

### CODE REVIEW

**Comment by timstclair** [comment]
I can't follow the logic here. I'll follow up with you tomorrow.

File: gubernator/templates/filtered_log.html [hunk 1]
```diff
@@ -26,7 +26,13 @@
 		<hr>
 		<label><input type="checkbox" name="UID"{% if uid %} checked {% endif %}> Highlight Pod UID: <input type="text" name="poduid" value="{{objref_dict['UID']}}" style="width: 300px;"></label><br>
 		<label><input type="checkbox" name="Namespace"{% if namespace %} checked {% endif %}> Highlight Namespace: <input type="text" name="ns" value="{{objref_dict['Namespace']}}" style="width: 300px;"></label><br>
-		<label><input type="checkbox" name="ContainerID"{% if containerID %} checked {% endif %}> Highlight ContainerID: <input type="text" name="cID" value="{{objref_dict['ContainerID']}}" style="width: 300px;"></label>
+		<label><input type="checkbox" name="ContainerID"{% if containerID %} checked {% endif %}> Highlight ContainerID: <input type="text" name="cID" value="{{objref_dict['ContainerID']}}" style="width: 300px;"></label><br>
+		Add filter: <input type="text" name="others"><br>
+		% for other in others|sort
+			% if other
+				<label><input type="checkbox" name="others" value={{other}} {% if other in others %} checked {% endif %}> {{other}}</label><br>
```
Comment (timstclair) [inline gubernator/templates/filtered_log.html]: nit: quotes around `{{other}}` in value

**Comment by timstclair** [comment]
LGTM

**Comment by mnshaw** [comment]
@rmmh  
Can this get merged?

**Pull request merged**